Documentation/CodingGuidelines/CglPhp/PhpFileFormatting/PhpSyntaxFormatting.rst

```diff
@@ -213,19 +212,19 @@ Assignment in conditions should be avoided. However if it makes sense
 to do an assignment in a condition, it should be surrounded by the
 extra pair of brackets. Example::
 
-   if (($fields = $GLOBALS['TYPO3_DB']->sql_fetch_assoc($res))) {
+   if (($fields = $this->getFields())) {
```

Comment:
why should it be better to use if ($variable) instead of if($variable !== false) ? the later would be type save.

Reply:
Also one pair of braces can be dropped here.

Reply:
@mbrodala The initial example was that way, but it does not make sense to me either. I will drop it. 


---

@linawolf 

> why should it be better to use if ($variable) instead of if($variable !== false) ? the later would be type save.

Don't know. In my opinion it depends, but a more experienced core developer could answer. (I would guess it does not provide much extra value in most cases and makes code more readable).

----

If there is more that should be changed in CGL, maybe we can do that in a separate patch.